### PR TITLE
Release 2.3.0

### PR DESCRIPTION
**Pull request by robertjd**
See changelog and upgrade guide

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -30,6 +35,25 @@ Many fixes for the registration field configuration:
 
 Please see the Registration section of this documentation for more information.
 
+Several bug fixes:
+
+- The `spaRoot` option was not observed by the change password route, so you
+  would get the standard HTML page and not your angular application.
+
+- On logout, access tokens and refresh tokens are now revoked via the Stormpath
+  REST API (this was not the case before - the would not be revoked).
```
Comment (rdegges) [inline docs/changelog.rst]: *the token

**Comment by rdegges** [comment]
You need to add in the upgrading guide docs -- this only has the changelog stuff.

**Comment by robertjd** [comment]
Upgrade doc already merged

**Comment by rdegges** [comment]
+1

**Pull request merged**